pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -83,7 +83,8 @@ func (ctrl *csiSnapshotSideCarController) syncContent(content *crdv1.VolumeSnaps
 		return false, nil
 
 	}
-	if content.Spec.Source.VolumeHandle != nil && content.Status == nil {
+	_, groupSnapshotMember := content.Labels[utils.VolumeGroupSnapshotHandleLabel]
+	if content.Spec.Source.VolumeHandle != nil && content.Status == nil && !groupSnapshotMember {
```

Comment:
Can you add a comment here to say that we'll only call the CSI driver to create snapshot if it is a dynamic provisioning for an independent snapshot or if the snapshot belongs to a group.